### ISSUE TITLE
Snort-3.1.5.0 shows huge outstanding packet count sporadically

### ISSUE THREAD

**Issue by redbaron4** [snort3/snort3#196]
I'm testing Snort3-3.1.5.0 on live traffic. The following entries are there in the `perf_monior_base.csv` file

```
#timestamp,daq.received,daq.analyzed,daq.dropped,daq.filtered,daq.outstanding,daq.rx_bytes
1624429932,18071616,13864997,0,4206615,4,14226412370
1624430232,18211875,14057686,489037,4154190,18446744073709551615,14708198265
1624430532,15103097,11048549,0,4054551,18446744073709551613,11081479729
1624430832,14610436,10537257,0,4073155,24,9913056502
1624431132,14971673,11114914,0,3856783,18446744073709551592,10304471375
1624431432,14299089,10832519,0,3466567,3,11620596471
1624431732,14337168,10568187,0,3768981,0,9766524798
1624432032,14878056,11064689,0,3813370,18446744073709551613,9890305603
1624432332,14386043,10589988,0,3796014,41,10169291105
1624432632,13620196,9813641,0,3806596,18446744073709551575,8324864163
1624432932,16634149,12361253,0,4272896,0,11708545429
1624433232,15621453,11446369,7724,4175080,4,10342789639
```

The second last column is `daq.outstanding` and it sometimes has an apparently nonsensical large value which looks suspiciously like the maximum (or thereabouts) of a 64-bit unsigned integer.

**Comment by ma-dagon** [comment]
Thank you for reporting the issue. We are looking into it.

**Comment by Normandez** [comment]
@redbaron4 
Thank you for reporting the issue. It has been fixed.
The fix commit will be in the master branch soon.

**Comment by redbaron4** [comment]
Unfortunately, this does appear to have been fixed.

As of 3.3.1, we still get huge values of outstanding

```
#timestamp,daq.received,daq.analyzed,daq.dropped,daq.filtered,daq.outstanding,daq.rx_bytes,detection.alerts
1723628859,2214295,2214302,0,0,18446744073709551609,1971797285,4
1723632159,1729790,1729767,0,0,16,1586471190,1
```

**Comment by redbaron4** [comment]
@Priyanka-Gurudev The issue seems to have been resolved when tested with Snort-3.3.4.0
